models/migrations/v57.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"code.gitea.io/gitea/modules/setting"
+	"fmt"
+	"github.com/go-xorm/xorm"
+	"time"
+)
+
+func addIssueDependencies(x *xorm.Engine) (err error) {
```

Comment:
There's no migration for The update to `Comment` 🤔

Reply:
Oups. Added one 😬